### DISCUSSION TITLE
Tooltip when hovering over shortcut

### DISCUSSION THREAD

**Discussion by jerryii1977** [Hofknecht/SystemTrayMenu#504] [Ideas]
I Love the app! I did have one request. When using a folder toolbar before, I could hover over a shortcut, and it would display the comment field as a tooltip. Can this functionality be added here?

**Comment by Hofknecht** [comment]
@jerryii1977 
Thanks for the feedack!
Thanks for the suggestion!
We'll look at it again later and probably only implement it in v2.

Tooltip was also mentioned here https://github.com/Hofknecht/SystemTrayMenu/issues/495, but for filename.
i can also imagine, e.g. to show the complete filenamepath or maybe if it is a shortcut then even to show the target of the shortcut. 
Maybe also possible in a extra dialog, we want maybe here also a extra dialog for jpg https://github.com/Hofknecht/SystemTrayMenu/issues/473

All these possiblities should be configurable
